config/config.go

```diff
@@ -958,7 +958,7 @@ func DefaultConsensusConfig() *ConsensusConfig {
 		TimeoutPrevoteDelta:         500 * time.Millisecond,
 		TimeoutPrecommit:            1000 * time.Millisecond,
 		TimeoutPrecommitDelta:       500 * time.Millisecond,
-		TimeoutCommit:               1000 * time.Millisecond,
+		TargetRoundDuration:         3500 * time.Millisecond,
 		SkipTimeoutCommit:           false,
 		CreateEmptyBlocks:           true,
 		CreateEmptyBlocksInterval:   0 * time.Second,
```

Comment:
[question] should this be 15 seconds given that's our desired block time?

```suggestion
		CreateEmptyBlocksInterval:   15 * time.Second,
```

Reply:
see above comment, we should set this in the app for a few different reasons. One is simply to avoid fixing all the tests that break and causing a huge diff

Reply:
imo no, at least not atm, as that is an optimization to avoid empty, and complicates timings. instead, I think we should focus on finding a solution that works for all blocks, such as setting the TargetRoundDuration to 15s (in the app ofc). If a tx appears in the mempool in that period of time, then it will simply be included in the next block

Reply:
I believe `CreateEmptyBlocksInterval` refers to how long to wait without any transactions in the mempool before going ahead and forming that empty block. Which means if transactions arrive, the block will be constructed far sooner than 15 seconds and it also does not consider how long it takes for consensus to be reached on that block